src/frontend/scss/_fonts.scss

```
@@ -1,32 +1,31 @@
 @font-face {
   font-family: 'Exo 2';
-  src: url('/static/richie/fonts/exo2-medium-webfont.woff2') format('woff2'),
-    url('/static/richie/fonts/exo2-medium-webfont.woff') format('woff');
+  src: url('/static/fonts/exo2-medium-webfont.woff2') format('woff2'),
```

My 2 cents:

1. Why not moving Django statics to `src/richie/apps/core/static/richie/{fonts,images}` to be more consistent with front-end build targets (css+js)?
2. What about using a relative path as it's done for the icomoon font?
```suggestion
$richie-fonts-path: '../fonts' !default;
// [...]
  src: url('#{$richie-fonts-path}/exo2-medium-webfont.woff2') format('woff2'),
```

---

@jmaupetit Good idea. I did it. Check it out!